config-builder/src/main/java/com/ca/apim/gateway/cagatewayconfig/bundle/builder/EntityBuilderHelper.java

```diff
@@ -61,4 +62,8 @@ static Entity getEntityWithMappings(final String type, final String path, final
         }
         return entity;
     }
+
+    static Entity getEntityWithDeleteMappings(final Entity entity){
+        return getEntityWithMappings(entity.getType(), entity.getName(), entity.getId(), null, MappingActions.DELETE, entity.getMappingProperties());
```

Comment:
This will have a conflict with PR #227 as I refactored `Entity.java`. Let #227 get merged first.